Source/Scene/Expression.js

```diff
@@ -1,26 +1,75 @@
 /*global define*/
 define([
+       '../Core/Color',
+       '../Core/defined',
        '../Core/defineProperties',
-       '../ThirdParty/jsep',
+       '../ThirdParty/jsep'
     ], function(
+        Color,
+        defined,
         defineProperties,
         jsep) {
     "use strict";
 
+    var LITERAL = 0;
```

Comment:
Put these in an "enum" (in this file, for now) like https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/TileState.js.